src/consensus/pbft/libbyz/New_principal.h

```diff
@@ -20,11 +20,8 @@ struct New_principal_rep : public Message_rep
   uint32_t ip_len;
   char ip[32];
 
-  uint32_t pubk_sig_len;
-  char pubk_sig[128];
-
-  uint32_t pubk_enc_len;
-  char pubk_enc[128];
+  uint32_t cert_len;
+  char cert[128];
```

Comment:
I'm not sure where this is used but 128 seems very small for a cert?

Reply:
Yes, I think this would ideally be a MAX_CERT_something coming from perhaps mbedtls.

Reply:
tls::Cert?

Reply:
Yes thanks, this was done via search/rename and I didn't notice it.